backend/hitas/views/apartment.py

```diff
@@ -831,6 +831,7 @@ def download_latest_unconfirmed_prices(self, request, **kwargs) -> Union[HttpRes
             "apartment": apartment_data,
             "additional_info": request.data.get("additional_info", ""),
             "surface_area_price_ceiling": sapc.value,
+            "old_hitas_ruleset": apartment.building.real_estate.housing_company.financing_method.old_hitas_ruleset,
```

Comment:
We have a small shortcut for this: `apartment.housing_company.financing_method.old_hitas_ruleset`
Even better, we could have a property in apartment model for this:
```py
@property
def new_hitas_rules():
    return apartment.completion_date >= datetime.date(2011, 1, 1) and not apartment.housing_company.financing_method.old_hitas_ruleset
```

Reply:
You wouldn't want to include the date in determining the ruleset, only the financing method counts, even if we don't actually have any pre-2011 housing companies with the new rulesets. That said, I think we should add the property.